core/src/Cuda/Kokkos_Cuda_Instance.cpp

```diff
@@ -213,7 +213,10 @@ void CudaInternal::print_configuration(std::ostream &s) const {
       << prop.major << "." << prop.minor
       << ", Total Global Memory: " << human_memory_size(prop.totalGlobalMem)
       << ", Shared Memory per Block: "
-      << human_memory_size(prop.sharedMemPerBlock);
+      << human_memory_size(prop.sharedMemPerBlock)
+      << ", Can access system allocated memory: " << prop.pageableMemoryAccess
+      << ", Via Address Translation Service: "
```

Comment:
I find Address Translation Services pretty cryptic even if it's accurate, maybe
```suggestion
      << ", via host pages: "
```

Reply:
The reasoning was, that this can be used as a seed for a google search if questions arise. For `host pages` it would be a diffuse search.